code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -484,18 +484,14 @@
 
 	return ..()
 
-/obj/item/weapon/storage/verb/quick_empty(var/turf/target)
+/obj/item/weapon/storage/verb/quick_empty()
```

Comment:
this was done intentionally, so boxes could be emptied when clicking on turfs
remove this

Reply:
alright so turns out the other guy had no idea that you _could_ empty quick_empty capable items by clicking on turfs, he said he'll get back to me on that eventually

Reply:
trying to empty contents onto the floor with the verb opens a menu that lets you drop stuff onto turfs in viewrange which is something that should be addressed though